### PR TITLE
Improve React Redirections [BTRX-637]

### PR DESCRIPTION
**Pull request by wloforteBroad**
## Addresses
https://broadinstitute.atlassian.net/browse/BTRX-637

## Changes
Include a high level summary of the changes and what they are intended for.
- Remove href to use react router instead

## Testing
Navigate all around the application and make sure all redirections are working as expected.

- Note: PR from Veronica, she is having issues with her GitHub account

---

- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Get a thumb from Belatrix
- [x] **Submitter**: Get a thumb from @rushtong
- [x] **Submitter**: Merge to develop using github's "Squash and Merge" feature
- [x] **Submitter**: Delete branch after merge

### CODE REVIEW

File: src/main/webapp/reviewCategories/ReviewCategories.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import React, { Component } from 'react';
-import { a, div, h1, hh, span } from 'react-hyperscript-helpers';
+import { a, div, h1, hh, span, h } from 'react-hyperscript-helpers';
 import { Reports } from '../util/ajax';
 import { TableComponent } from '../components/TableComponent';
 import { CATEGORY_SORT_NAME_INDEX, styles } from '../util/ReportConstants';
 import { TABLE_ACTIONS } from '../util/TableUtil';
 import { handleRedirectToProject } from '../util/Utils';
```
Comment (rushtong) [inline src/main/webapp/reviewCategories/ReviewCategories.js]: This looks unused now

File: src/main/webapp/reviewCategories/ReviewCategories.js [hunk 2]
```diff
@@ -67,6 +75,13 @@ const columns = [
   }
 ];
 
+export const linkFormatter = (row)  => {
```
Comment (rushtong) [inline src/main/webapp/reviewCategories/ReviewCategories.js]: Is this used?

File: grails-app/controllers/org/broadinstitute/orsp/api/DataUseController.groovy [hunk 3]
```diff
@@ -80,19 +80,6 @@ class DataUseController extends AuthenticatedController {
         }
     }
 
-    def saveSdul() {
-        try {
-            DataUseRestriction restriction = dataUseLetterService.getRestrictionFromParams(request.JSON)
```
Comment (rushtong) [inline grails-app/controllers/org/broadinstitute/orsp/api/DataUseController.groovy:85]: `dataUseLetterService` is no longer used in this file.

**Review by rushtong — APPROVED** [review]
Minor suggestions inline - no need to re-review if you want to clean up the unused code.

**Review by LeoForconesi — APPROVED** [review]

**Pull request merged**